### DISCUSSION TITLE
Ethernet Shield support

### DISCUSSION THREAD

**Discussion by mattrob33** [Rahix/avr-hal#323] [Q&A]
I'm trying to use the Ethernet Shield rev2 hardware (and Ethernet library) with this. Is this supported already or is there a straightforward way to support it? There is an existing w5500 driver crate already, but I'm not sure if it's compatible with this project.

**Comment by Rahix** [comment]
Right now I do not know if anyone ever attempted this.  For sure I haven't seen any code so far.  That doesn't mean that it won't work.  If the w5500 driver uses `embedded-hal`, there is a good chance that you can connect it to `avr-hal` with ease.  Let me know how it goes.  If you hit any problems, me or someone else here might also be able to help, so do post them as well.
